parm/fv3lam_cmaq.xml-pmtf_only

```diff
@@ -0,0 +1,3423 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@hsinmulin-NOAA Please remove all of your testing files like this one from your PR.

Reply:
removed